### PR TITLE
feat(corev3): local debug

### PR DESCRIPTION
**Pull request by jayzhang**

### CODE REVIEW

File: packages/fx-core/src/common/local/projectSettingsHelper.ts [hunk 1]
```diff
@@ -12,20 +17,24 @@ export class ProjectSettingsHelper {
...
     );
 
-  public static includeFrontend = (projectSettings: ProjectSettings | undefined): boolean =>
-    !!(projectSettings?.solutionSettings as AzureSolutionSettings)?.activeResourcePlugins?.some(
-      (pluginName) => pluginName === ResourcePlugins.FrontendHosting
-    );
-
+  public static includeFrontend = (projectSettings: ProjectSettings | undefined): boolean => {
+    if (!projectSettings) return false;
+    const solutionSettings = projectSettings.solutionSettings as AzureSolutionSettings;
```
Comment (swatDong) [inline packages/fx-core/src/common/local/projectSettingsHelper.ts:22]: Will `solutionSettings` or `capabilities` be undefined?

Comment (jayzhang) [reply packages/fx-core/src/common/local/projectSettingsHelper.ts:22]: No, their are required

File: packages/fx-core/src/common/local/projectSettingsHelper.ts [hunk 2]
```diff
@@ -12,20 +17,24 @@ export class ProjectSettingsHelper {
...
+    return solutionSettings.capabilities.includes(TabOptionItem.id);
```
Comment (swatDong) [inline packages/fx-core/src/common/local/projectSettingsHelper.ts:23]: Consider backward compatibility - what if opening existing (before v3) project?
Is `solutionSettings.version` designed for determining whether to use `solutionSettings.activeResourcePlugins` or `solutionSettings.capabilities`?

Comment (jayzhang) [reply packages/fx-core/src/common/local/projectSettingsHelper.ts:23]: Yes, but a better way is to use capabilities for both versions. Capabilities are the single truth.

**Pull request closed without merging**